recipes/geoh5py/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "geoh5py" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geoh5py-{{ version }}.tar.gz
```

Comment:
Point to github repo instead of pypi.
Using main URL ? https://github.com/MiraGeoscience/geoh5py/archive/refs/heads/main.tar.gz

this way, we can wait for Conda-froge to be OK and adjust as necessary before publishing to pypi